CrackingTheCodingInterview/Chapter7MathematicsAndProbability/Operations.java

```diff
@@ -40,6 +40,9 @@ public static int abs(int a) {
 		else {
 			return a;
 		}
+		
+		//or
+		return Math.abs(a);
```

Comment:
Of course I agree with you. Sorry for adding incorrect code and thank you VERY much for your response.